specs/jsonschema-validation.md

```diff
@@ -880,7 +881,7 @@ draft-bhutton-json-schema-01, June 2022,
 
 ### Informative References
 
-#### [RFC4329] {#rfc4329}
+#### \[RFC4329\] {#rfc4329}
```

Comment:
Why does this need backslashes, but the `[Appendix]` on 899 doesn't?

Reply:
`[whatever]` needs backslashes because it clashes with link syntax. The slashes make it clear that those are literal characters and not a broken link declaration. `[Appendix]` is special placeholder used by the TableOfContents plugin. It gets converted to text before the linting occurs. In hindsight, I should have used a different placeholder than doesn't clash with link syntax. We still could make that change if we want.

Reply:
It's evidently confusing.  Let's look at an alternate syntax, please.

Reply:
I changed it to `%appendix%`